Resources/doc/index.md

```diff
@@ -10,6 +10,22 @@ of this bundle. For version `1.x`, please read:
 Also, you might be interested in this [UPGRADE
 guide](https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/UPGRADE.md).
 
+- [Installation](/blob/master/Resources/doc/index.md#installation)
```

Comment:
that won't work due to the way Github handles relative links when rendering doc (it treats them as inside the repo, so `/blob/master` should not be there).

and as these are only links inside the same page, you could even simplify it using `#installation` only

Reply:
You're right, thanks!